ftl/core/deck-config.ftl

```diff
@@ -38,7 +38,7 @@ deck-config-tab-description =
     - `Preset`: The limit is shared with all decks using this preset.
     - `This deck`: The limit is specific to this deck.
     - `Today only`: Make a temporary change to this deck's limit.
-deck-config-new-cards-ignore-review-limit = New cards ignore review limit
+deck-config-new-cards-ignore-review-limit = New cards ignore maximum limit
```

Comment:
```suggestion
deck-config-new-cards-ignore-review-limit = New cards ignore maximum
```

No strong feelings here, but what do you think about this instead?

Reply:
The app will be used by people with varied levels of product knowledge and English ability. I think 'limit' is clearer.